src/components/CircularProgress/CircularProgress.js

```diff
@@ -9,17 +9,28 @@ import PropTypes from 'prop-types';
  * import { CircularProgress } from 'weaveworks-ui-components'
  *
  * React.render(
- *   <CircularProgress size={45} />
+ *   <div>
+ *     <CircularProgress size={35} />
+ *     <CircularProgress inline /> Inline
+ *     <CircularProgress center />
+ *   </div>
```

Comment:
The example looks tiny bit like something's broken:

![image](https://user-images.githubusercontent.com/1216874/34828201-12ef39c0-f6dd-11e7-9a17-cacd4dd08099.png)

Maybe adding some paddings/explanations in between would help (e.g. like in a Time Travel example).

Reply:
Totally agree. Created reusable components and slightly modified them (left-aligned headings).